Extensions/Video/videoruntimeobject-pixi-renderer.js

```diff
@@ -149,13 +143,15 @@ gdjs.VideoRuntimeObjectPixiRenderer.prototype.pause = function() {
 
 /**
  * Set the loop on video in renderer
+ * @param {boolean} bool The new state of looped.
  */
 gdjs.VideoRuntimeObjectPixiRenderer.prototype.setLoop = function(bool) {
```

Comment:
nitpicking: avoid naming something with just the type ("bool"). Instead, give a name that convey the meaning of this boolean: `enable`
(as this is a boolean that enable or disable looping).